plugins/inputs/procstat/procstat.go

```diff
@@ -92,7 +95,14 @@ func (_ *Procstat) Description() string {
 	return "Monitor process cpu and memory usage"
 }
 
+type PIDS_TAGS_ERR_GROUP struct {
```

Comment:
I think that putting PID and tags together in a struct makes sense, but it might be more idiomatic to keep error separate.

Reply:
Looking more into this, I see why it is structured this way.